scripts/install

```diff
@@ -30,169 +30,123 @@ path_url=$YNH_APP_ARG_PATH
 
-## Start php7
-systemctl enable php7.2-fpm
-systemctl start php7.2-fpm
+ynh_install_app_dependencies $pkg_dependencies
 
 #=================================================
 # CREATE A MYSQL DATABASE
 #=================================================
-# If your app uses a MySQL database, you can use these lines to bootstrap
-# a database, an associated user and save the password in app settings
+ynh_script_progression --message="Creating a MySQL database..." --weight=1
 
-db_name=$(ynh_sanitize_dbid $app)
-ynh_app_setting_set $app db_name $db_name
-ynh_mysql_setup_db $db_name $db_name
+db_name=$(ynh_sanitize_dbid --db_name=$app)
+db_user=$db_name
+ynh_app_setting_set --app=$app --key=db_name --value=$db_name
+ynh_mysql_setup_db --db_user=$db_user --db_name=$db_name
 
 #=================================================
 # DOWNLOAD, CHECK AND UNPACK SOURCE
 #=================================================
+ynh_script_progression --message="Setting up source files..." --weight=1
 
-ynh_app_setting_set $app final_path $final_path
+ynh_app_setting_set --app=$app --key=final_path --value=$final_path
 # Download, check integrity, uncompress and patch the source from app.src
-ynh_setup_source "$final_path"
+ynh_setup_source --dest_dir="$final_path"
 
 #=================================================
 # NGINX CONFIGURATION
 #=================================================
+ynh_script_progression --message="Configuring NGINX web server..." --time --weight=1
 
-# Create a dedicated nginx config
+# Create a dedicated NGINX config
 ynh_add_nginx_config
 
 #=================================================
 # CREATE DEDICATED USER
 #=================================================
+ynh_script_progression --message="Configuring system user..." --time --weight=1
 
 # Create a system user
-ynh_system_user_create $app
+ynh_system_user_create --username=$app
 
 #=================================================
 # SPECIFIC SETUP
 #=================================================
 
-# create a user
-email=$(ynh_user_get_info $admin 'mail')
-
 # setup application config
-cp -a ../conf/.env.example $final_path/.env
+cp -a ../conf/.env $final_path/.env
+
 cp -a ../conf/wings.service /etc/systemd/system/wings.service
 
 #=================================================
 # MODIFY A CONFIG FILE
 #=================================================
 
 #ynh_replace_string "random_key" "$random_key" "$final_path/.env"
-ynh_replace_string "yunouser"   "$db_name"    "$final_path/.env"
-ynh_replace_string "yunopass"   "$db_pwd"     "$final_path/.env"
-ynh_replace_string "yunobase"   "$db_name"    "$final_path/.env"
-ynh_replace_string "yunomail"   "$email"      "$final_path/.env"
-ynh_replace_string "yunodomain" "$domain"     "$final_path/.env"
+ynh_replace_string "__DB_USER__"   "$db_name"    "$final_path/.env"
+ynh_replace_string "__DB_PASS__"   "$db_pwd"     "$final_path/.env"
+ynh_replace_string "__DB_NAME__"   "$db_name"    "$final_path/.env"
```

Comment:
You should use https://yunohost.org/fr/packaging_apps_helpers?q=%2Fpackaging_apps_helpers
`ynh_replace_string --match_string=match_string --replace_string=replace_string --target_file=target_file`